src/components/BobaContainer/bobalist.jsx

```diff
@@ -0,0 +1,14 @@
+const bobalist = [
```

Comment:
Camel-case `bobaList`

Reply:
updated.

Reply:
Please mark resolved conversation if you validated as completed.